pkg/piecestore/pstore.go

```diff
@@ -20,86 +19,70 @@ var (
 	FSError  = errs.Class("fsError")
 )
 
-// PathByHash -- creates datapath from hash and dir
-func PathByHash(hash, dir string) (string, error) {
-	if len(hash) < 20 {
-		return "", ArgError.New("Invalid hash length")
+// PathByID -- creates datapath from id and dir
+func PathByID(id, dir string) (string, error) {
+	if len(id) < 20 {
```

Comment:
Should we make this 20 a const somewhere ? Would we ever want the id to be less than 20 ?

Reply:
done